vmdb/db/migrate/20150401223517_container_nodes_hardware.rb

```diff
@@ -0,0 +1,8 @@
+class ContainerNodesHardware < ActiveRecord::Migration
+  def change
+    add_column :hardwares, :container_node_id, :bigint
+    add_column :container_nodes, :infra_machine_id, :string
+    add_column :container_nodes, :machine_id, :string
+    add_column :container_nodes, :system_uuid, :string
```

Comment:
@Fryguy any insight about these 3? system_uuid is the bios system uuid, machine_id is the id generated (and persisted on disk) by the operating system, and infra_machine_id is a special id assigned by the infrastructure (e.g. rhev, openstack, etc.) to identify the node.

Is it worth having these in another table or ContainerNodes is good enough?